test/e2e/spec/eid_oidc_provider/auth_spec.js

```diff
@@ -0,0 +1,108 @@
+describe("eId OpenId: The authorization endpoint", function() {
+    
+    var chakram = require('/usr/local/lib/node_modules/chakram/lib/chakram.js');
+    var expect = chakram.expect;
+    var URL = 'https://' + process.env.BOILERPLATE_DOMAIN + '/api/openid';
```

Comment:
Use URL + `authorize`.

Reply:
Possible take advantage of chakram's baseurl setting? It could save you some copy & paste.